data/avro-schema/src/main/resources/avro/net/corda/data/flow/FlowTimeout.avsc

```diff
@@ -0,0 +1,20 @@
+{
+  "type": "record",
+  "name": "FlowTimeout",
+  "namespace": "net.corda.data.flow",
+  "doc": "Represents of a flow timeout instruction.",
+  "fields": [
+      {
+        "name": "checkpointStateKey",
```

Comment:
As I understand, this is the flow ID, but I thought maybe it's worth being specific in case this changes/evolves.

Reply:
Think there's no harm in having it.